### ISSUE TITLE
Release a new version of `tracing-log`

### ISSUE THREAD

**Issue by koute** [tokio-rs/tracing#1664]
It'd be nice to have the current `v0.1.x` branch pushed out to `crates.io` (as `0.1.3`?).

(If there's anything that still needs to be done before the new version can be uploaded I'd be happy to help out.)

**Comment by koute** [comment]
Since https://github.com/tokio-rs/tracing/pull/1670 seems to be stalled could we maybe get a new release of `tracing-log` as-is anyway please? (:

**Comment by hawkw** [comment]
Yes, sorry for the delay!

I'd also like to give #1670 another review, as well --- sorry about that; I've been busy with a lot of other things.

**Comment by koute** [comment]
Gentle reminder. (:

**Comment by bkchr** [comment]
@hawkw is there any chance we can get this released? Please :)

**Comment by hawkw** [comment]
Fixed by #2086. Sorry for the delay!

**Comment by bkchr** [comment]
@hawkw ty :)
